css/pages/auth.css

```diff
@@ -0,0 +1,95 @@
+main {
+  position: relative;
+}
+
+.auth__container {
+  height: auto;
+  width: 35rem;
+  padding: 2rem;
+  background-color: #f5f3f3;
+  font-size: 1.2rem;
+  position: absolute;
```

Comment:
make a helper class for absolute-container you can use this code another component as well

Reply:
ok will do that.